src/components/Validations/ValidationSummary.tsx

```diff
@@ -9,6 +9,7 @@ interface Props {
   id: string;
   errors: number;
   warnings: number;
+  objectCount?: number;
```

Comment:
I wonder why it is optional, is it that when there's 0 object, it's undefined instead of 0 ? In that case I think it would be better to explicitly initialize to 0 server-side, and not make it optional here

Reply:
I am using the optional because this component is used besides the overview and graph page. It is used in the Istio Config list. It is kind of a flag for showing one thing on the tooltip or another.
Perhaps, I could add one extra prop in the component to be more explicit and more readable.
The behavior is: when there is ObjectCount then the tooltip subtitle says the number of analyzed objects. Otherwise, it shows the aggregation severity (highest one).

Reply:
fair enough :)